diff -Naur ser-0.8.10/Makefile.defs ser-0.8.10-p/Makefile.defs
--- ser-0.8.10/Makefile.defs	Mon Jun 23 11:20:43 2003
+++ ser-0.8.10-p/Makefile.defs	Mon Jun 23 11:20:58 2003
@@ -135,11 +135,12 @@
 	 -DOS='"$(OS)"' -DCOMPILER='"$(CC_VER)"' -D__CPU_$(ARCH)\
 	 -DCFG_DIR='"$(cfg-target)"'\
 	 -DPKG_MALLOC \
-	 -DSHM_MEM  -DSHM_MMAP \
+	 -DSHM_MEM \
 	 -DADAPTIVE_WAIT -DADAPTIVE_WAIT_LOOPS=1024 \
 	 -DDNS_IP_HACK \
 	 -DUSE_IPV6 \
 	 -DDBG_QM_MALLOC \
+	 # -DSHM_MMAP \
 	 #-DF_MALLOC \
 	 #-DNO_DEBUG \
 	 #-DNO_LOG
diff -Naur ser-0.8.10/mem/shm_mem.c ser-0.8.10-p/mem/shm_mem.c
--- ser-0.8.10/mem/shm_mem.c	Mon Nov  4 18:05:32 2002
+++ ser-0.8.10-p/mem/shm_mem.c	Mon Jun 23 11:40:18 2003
@@ -204,7 +204,11 @@
 #ifdef SHM_MMAP
 	if (shm_mempool && (shm_mempool!=(void*)-1)){
 #else
-	if ((shm_shmid!=-1)||(shm_semid!=-1)||(shm_mempool!=(void*)-1)){
+	if ((shm_shmid!=-1)
+#ifndef FAST_LOCK
+		||(shm_semid!=-1)
+#endif
+		||(shm_mempool!=(void*)-1)){
 #endif
 		LOG(L_CRIT, "BUG: shm_mem_init: shm already initialized\n");
 		return -1;
