--- ser-0.8.10/cfg.y.orig	Sat Nov  2 02:35:08 2002
+++ ser-0.8.10/cfg.y	Thu Jul  3 16:52:38 2003
@@ -151,9 +151,9 @@
 
 
 /*non-terminals */
-%type <expr> exp, exp_elem /*, condition*/
-%type <action> action, actions, cmd, if_cmd, stm
-%type <ipaddr> ipv4, ipv6, ip
+%type <expr> exp exp_elem /*, condition*/
+%type <action> action actions cmd if_cmd stm
+%type <ipaddr> ipv4 ipv6 ip
 %type <ipnet> ipnet
 %type <strval> host
 %type <strval> listen_id
@@ -582,7 +582,7 @@
 						  	$$[strlen($1)+1+strlen($3)]=0;
 						  }
 						  free($1); free($3);
-						};
+						}
 	| host DOT error { $$=0; free($1); yyerror("invalid hostname"); }
 	;
 
